src/MercadoPago/Client/Invoice/InvoiceClient.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace MercadoPago\Client\Invoice;
+
+use MercadoPago\Client\MercadoPagoClient;
+use MercadoPago\Core\MPRequestOptions;
+use MercadoPago\Exceptions\MPApiException;
+use MercadoPago\MercadoPagoConfig;
+use MercadoPago\Net\HttpMethod;
+use MercadoPago\Net\MPSearchRequest;
+use MercadoPago\Resources\Invoice;
+use MercadoPago\Resources\InvoiceSearch;
+use MercadoPago\Serialization\Serializer;
+
+/** Client responsible for performing invoice actions. */
+class InvoiceClient extends MercadoPagoClient
+{
+    private static $URL_WITH_ID = "/authorized_payments/%s";
```

Comment:
Why it isn't a constant?

Reply:
Since the idea is that these values are immutable, it really makes sense that we use constants. Good idea!